angular-frontend/angular.json

```diff
@@ -71,9 +72,6 @@
               "browserTarget": "angular-frontend:build:development"
             }
           },
-          "options": {
```

Comment:
**[question]**: has the proxy conf gone away?

Reply:
Ah no. I was trying to revert something I had done for trouble shooting. Thanks!